src/android/CHIPTool/app/build.gradle

```diff
@@ -18,6 +18,23 @@ android {
         // NOTE: This build assumes CHIP was configured and built for armeabi-v7a. Deal with
         // other archs later when build is sane.
         //  ndk {abiFilters 'armeabi-v7a'}
+
+        externalNativeBuild {
+            // For ndk-build, instead use the ndkBuild block.
+            cmake {
+                arguments "-DBUILD_SHARED_LIBS=ON",
+                          "-DOT_COMM_ANDROID=ON",
+                          "-DOT_COMM_JAVA_BINDING=ON",
+                          "-DOT_COMM_APP=OFF",
+                          "-DOT_COMM_CCM=OFF",
+                          "-DOT_COMM_COVERAGE=OFF",
+                          "-DOT_COMM_TEST=OFF",
+                          "-DOT_COMM_JAVA_BINDING_OUTDIR=$projectDir/src/main/java"
+                // Specifies the library and executable targets from your CMake project
+                // that Gradle should build.
+                targets "commissioner-java", "event_core_shared", "event_pthreads_shared"
+            }
```

Comment:
We are using automake build system to get the .so and .jar artifacts to be included in Android apps. 
Let's have a single system and avoid adding in cmake along with automake.

Reply:
Will do.